readme.txt

```diff
@@ -2,6 +2,10 @@
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Fix - Added IE11 compatibility for the toggles styles. [ET-865]
```

Comment:
Lets try to make it more clear to which elements this applies to, maybe:

```suggestion
* Fix - Added IE11 compatibility for the toggles styles using `tribe-common-form-control-toggle` HTML class. [ET-865]
```

Reply:
Wouldn't it be CSS class in that case?